tests/Functional/Helper/CacheHelper.php

```
@@ -14,24 +14,49 @@
-     */
-    class CacheHelper implements CacheInterface
-    {
-        use CacheHelperV7;
+/**
+ * @internal
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CacheHelper implements CacheInterface
+{
+    public function get(string $key, mixed $default = null): mixed
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)